starfish/spots/_target_assignment/_base.py

```diff
@@ -30,7 +22,7 @@ def _cli_run(cls, ctx, instance):
         assigned.to_netcdf(os.path.join(output))
 
     @staticmethod
-    @click.group(COMPONENT_NAME)
+    @click.group("TargetAssignment")
```

Comment:
```suggestion
    @click.group("AssignTargets")
```

(and all dependent refactoring).

This is action oriented like the rest of the names.

Reply:
done.